src/olympia/reviewers/management/commands/review_reports_sql/addon/breakdown.sql

```diff
@@ -0,0 +1,26 @@
+SELECT 'All Reviewers' AS `Group`,
+       FORMAT(SUM(aa.weight), 0) AS `Total Risk`,
+       FORMAT(AVG(aa.weight), 2) AS `Average Risk`,
+       FORMAT(COUNT(*), 0) AS `Add-ons Reviewed`
+FROM editors_autoapprovalsummary aa
+JOIN reviewer_scores rs ON rs.version_id = aa.version_id
+WHERE DATE(rs.created) BETWEEN @WEEK_BEGIN AND @WEEK_END
+  AND user_id <> 4757633
+  AND rs.note_key IN (10, 12, 20, 22, 30, 32, 50, 52, 102, 103, 104, 105)
```

Comment:
Need a comment regarding what these values correspond to (same thing for the other files)